Automattic/woocommerce-payments
# Update naming convention from "platform checkout" to "woopay"

Fixes https://github.com/Automattic/woocommerce-payments/issues/6070

#### Changes proposed in this Pull Request

- Change all "platform checkout" texts to "woopay" according to their forms.
- Only change variables, parameters (used internally in merchant/client sites, e.g. the front-end code), const names. 
- DO NOT change option name keys (saved in the wp_options), params and endpoints to interact with the server.









#### Testing instructions

- All PHP and JS tests are passed. 
- I would suggest looking at my inline comments and verify they're all reasonable.
- Go through all changes and spot out non-reasonable changes.
- (Maybe) Run this through weekly WooPay tests by GlobalStep too.




*

-------------------


**Post merge**
